spec/unit/models/uprn_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe Uprn, type: :model do
```

Comment:
I think this is fixed, although it looks a bit spaced out because of the "space between test setup and expect" I've been doing, which in this simple file with lots of 2-line tests makes it look very whitespacey.